marklogic-data-hub-central/ui/src/config/tooltips.config.tsx

```diff
@@ -43,6 +43,7 @@ const ModelingTooltips = {
   nameEntityProperty: 'The name of this entity property. ' +
     'Names must start with a letter and can contain letters, numbers, hyphens, and underscores.',  /* intended dupe: all names */
   descriptionEntityProperty: 'A description of this entity property.',
+  reservedEntityNames: ' is not an allowed entity name. Choose another.',
```

Comment:
@devinnicholson, if you haven't already, can you have docs review this (I'd tag Al but I don't see his GitHub). I suspect they'll want to massage the wording.

Reply:
Al confirmed wording is OK.